### ISSUE TITLE
Falcine sinus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/uberon#3211]
**Preferred term label:**
Falcine sinus

**Synonyms**

**Definition (free text, please give PubMed ID)**
The falcine sinus is a normal fetal structure located within the [falx cerebri](https://radiopaedia.org/articles/falx-cerebri?lang=us) that drains the deep cerebral venous system to the [superior sagittal sinus](https://radiopaedia.org/articles/superior-sagittal-sinus?lang=us). Normally, it involutes after birth. (See [radiopaedia](https://radiopaedia.org/articles/falcine-sinus-1))

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0736-9199

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000295
- label: Persistent falcine venous sinus
- parents: [Abnormal cerebral venous sinus morphology (HP:0033723)](https://hpo.jax.org/app/browse/term/HP:0033723)
- pmid: PMID:19779000
- def: A falcine sinus that does not involute but persists beyond birth is defined as a persistent falcine sinus. In the 20-mm embryo stage, the primitive falx cerebri contains the sagittal plexus, a meshwork of anastomotic venous channels from which the superior sagittal sinus and straight sinus are thought to develop. A persistent falcine sinus represents a persistent channel of the caudal anastomotic loops of the sagittal plexus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9866
- synonyms:
